### PR TITLE
2048 Game

### LINKED ISSUE #908: 2048 Game (Python)
**Is your feature request related to a problem? Please describe.**
Why not we can make 2048 game by our self as we enjoy a lot playing this game.

**Describe the solution you'd like**
Here using tkinter and random we will generate blocks of different colour and number and as per our move we will add the numbers
and as all know the aim of game is to reach 2048 number .

### PR DESCRIPTION
**Pull request by neelshah2409**
## Explanation
  2048 a game developed in python programming Language using random module and tkinter module.
It is a single player sliding tile puzzle game
The user can keep playing till he gets a 2048 tile or all the grids are full and no grids can be merged

Fixes #908 

## Checklist

- [x] The PR title starts with "Fix #issue_number".  
- [x] My branch is up-to-date with the main branch.
- [x] The PR does not contain any unnecessary auto-generated code.
- [x] The PR does not break any part of the existing code.
- [x] The PR creator have carefully read & adher to the guidelines of the project.
- [x] The PR is raised after the issue has been assigned to the PR creator.

## Output
output_1(2048)
output_2(2048)

### CODE REVIEW

**Review by GowthamGoush — CHANGES_REQUESTED** [review]
Please consider removing the changes to the irrelevant readme.

**Comment by neelshah2409** [comment]
@GowthamGoush Sir I had updated my readme ,please see it.

File: Python/Job Finder Helper/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Job Finder Helper
```
Comment (GowthamGoush) [inline Python/Job Finder Helper/README.md:1]: Consider removing the changes in this readme.

**Review by GowthamGoush — COMMENTED** [review]
I've mentioned the required changes below.

**Comment by GowthamGoush** [comment]
Also, you may feel free to call me by my name.

**Comment by neelshah2409** [comment]
@GowthamGoush I had done the changes please review it.

**Comment by neelshah2409** [comment]
@GowthamGoush Please see this PR.

**Comment by GowthamGoush** [comment]
The changes are still reflected @neelshah2409. Try to copy/paste the exact data into that readme.

**Comment by neelshah2409** [comment]
> The changes are still reflected @neelshah2409. Try to copy/paste the exact data into that readme.

@GowthamGoush sir I am not getting still if you tell than I may make a new  PR and you can close if you tell . Please help.

**Comment by GowthamGoush** [comment]
Hey, @neelshah2409, no worries. All you have to do is remove any changes you've made to the `Python/Job Finder Helper/README.md`. You may reset the head or copy/paste the original copy, or create a new branch and push only those changes concerned with your script.

**Comment by GowthamGoush** [comment]
@neelshah2409, even deleting the readme will be reflected as change. Consider creating a new branch and pushing all the necessary changes into that.

**Comment by neelshah2409** [comment]
@GowthamGoush Sir please see if it is done or sir please close this PR I will make another for this please reply sir because today is last day.

**Pull request closed without merging**